docs/Error-Handling.md

```diff
@@ -0,0 +1,9 @@
+<h1 align="center">Fastify</h1>
+
+## Error Handling
+
+Uncaught errors are likely to cause memory leaks, file descriptor leaks and other major production issues ([see this])(http://npm.im/make-promises-safe). [Domains](https://nodejs.org/en/docs/guides/domain-postmortem/) were introduced to try fixing this issue, but they did not. Given the fact that it is not possible to process all uncaught errors in a sensible way, the best way to deal with them at the moment is to [crash](https://nodejs.org/api/process.html#process_warning_using_uncaughtexception_correctly). In case of promises, make sure to [handle](https://nodejs.org/dist/latest-v8.x/docs/api/deprecations.html#deprecations_dep0018_unhandled_promise_rejections) catch errors [correctly](https://github.com/mcollina/make-promises-safe).
```

Comment:
`([see this])(http://npm.im/make-promises-safe)` is malformed, you should correct the parenthesis.
https://github.com/fastify/fastify/blob/c2b473750dd2f4f01351c517a7053d46748599f7/docs/Error-Handling.md

Reply:
Removed the link since there is already a link to `https://github.com/mcollina/make-promises-safe` below